packages/@azure/servicebus/data-plane/test/stress/fixedNumberOfClientsSingleQueue.ts

```diff
@@ -0,0 +1,63 @@
+import { ServiceBusClient, SendableMessageInfo } from "../../lib";
+
+const connectionString = "";
+const queueName = "";
+
+const testDurationInMilliseconds = 60000 * 5 * 12 * 24 * 7; // 1 week
+
+const numOfClients = 10;
+
+let msgId = 1;
+
+let snapshotIntervalID: any;
+
+let isJobDone = false;
+
+async function main(): Promise<void> {
+  snapshotIntervalID = setInterval(snapshot, 5000); // Every 5 seconds
+  setTimeout(() => {
+    isJobDone = true;
+  }, testDurationInMilliseconds);
+
+  sendReceiveMessages();
+}
+
+async function sendReceiveMessages(): Promise<void> {
+  const ns = ServiceBusClient.createFromConnectionString(connectionString);
+
+  const clients = [];
+  for (let i = 0; i < numOfClients; i++) {
+    clients[i] = ns.createQueueClient(queueName);
+  }
+
+  try {
+    while (!isJobDone) {
+      for (let i = 0; i < numOfClients; i++) {
+        const sender = clients[i].createSender();
```

Comment:
We will have to pull this out as we can call `createSender()` only once if we don;t close the previous sender
Same for receiver